resources/public/api/conf/1.0/application.raml

```diff
@@ -70,7 +70,7 @@ uses:
                   value:
                     code: NOT_FOUND
 
-  /self-assessment:
+  /self-assessment/{matchId}:
     get:
       displayName: "Get an Organisation's Self Assessment details."
       description: "Get details such as SA Start date, tax solvency status and SA Returns."
```

Comment:
Remove capitalisation on [S]tart date and [R]eturns, and change SA -> self assessment.
(unless instructed otherwise by Lisa 🙂)